Source/bmalloc/Configurations/Base.xcconfig

```diff
@@ -168,3 +168,5 @@ WK_USER_LTO_MODE_thin = YES_THIN;
 WK_USER_LTO_MODE_none = NO;
 WK_USER_LTO_MODE_ = $(WK_DEFAULT_LTO_MODE);
 WK_DEFAULT_LTO_MODE = $(WK_USER_LTO_MODE_thin);
+
+OTHER_LDFLAGS = $(inherited) -Xlinker -no_inits;
```

Comment:
Do we need this? libbmalloc is built as static library, so we actually link things at JavaScriptCore.framework layer.